modules/ROOT/pages/extending-the-runtime-manager-agent.adoc

```diff
@@ -1,13 +1,18 @@
-= Extending the Runtime Manager Agent for Hybrid
+= Extending Runtime Manager Agent for Hybrid
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This topic contains an overview of how to extend the Runtime Manager agent. It provides instructions on implementing a complete service, external handler and internal hander using a method to revert a string.
+image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-active.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
+
+This topic contains an overview of how to extend Anypoint Runtime Manager agent. It provides instructions on implementing a complete service, external handler and internal hander using a method to revert a string.
```

Comment:
Global: Preference is to avoid self-referential text. So more like "You can extend Anypoint Runtime Manager by implementing a complete service...."

Not global: I am confused about whether the handlers (not hander in "internal hander") are part of implementing a complete service or other ways of extending RMA. Is using a method to revert a string how you implement a service? part of the handlers (and are those error handlers or something else)?

Reply:
oh, I see they are data handlers from text following. maybe make that explicit in this intro paragraph?